librb/include/rb_include.h

```diff
@@ -0,0 +1,44 @@
+#ifndef _RB_INCLUDE_H
+#define _RB_INCLUDE_H 1
+
+// This is exactly how they recommend doing it in the docs by Alan Smithee,
+// but the gcc docs on the other hand, with tongue in cheek, tell the tale
+// by metaphor: https://gcc.gnu.org/onlinedocs/cpp/Computed-Includes.html
+
+#include <RB_INC_ASSERT_H
```

Comment:
Eliminates the need to have individual #ifdef #include #endif lines all down the file when supporting multiple platforms, since the preprocessor only does one pass.